frontend/src/lib/stores.ts

```diff
@@ -2,11 +2,14 @@ import {
     readable, writable, type Readable, type Writable,
 } from "svelte/store";
 import type { Stock } from "$lib/types";
+import type { TickerDetails } from "./api/stocks";
 
 export const user: Writable<{ username: string } | null> = writable(null);
 
 export const currentStock: Writable<Stock> = writable({ ticker: "AAPL", name: "Apple Inc", exchange: "NASDAQ" });
 
+export const stocksDetails: Writable<Map<string, TickerDetails> | null> = writable(null);
```

Comment:
Is there any special handling we need to do when stocksDetails is null? If not, can simply initialise to an empty map.

Reply:
I thought that null would be more representative, as the Map itself of stocks is not loaded at all - so it rather specifies there isn't a Map at all rather than there is one but empty. Can change it if you feel otherwise, not a problem at all

Reply:
Try integrate it with E.g. holdings and see which one you prefer, having it empty reduces the number of cases we need to handle but if could be worth having the extra information :)

Reply:
Sure - but at least worth thinking about how it would be used.